<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="palyer1_Btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Switch player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="player2_Btn">
    <property name="geometry">
     <rect>
      <x>479</x>
      <y>80</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Switch player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sec150_Btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>120 sec</string>
    </property>
   </widget>
   <widget class="QPushButton" name="min5_Btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5 min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startGame_Btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START GAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopGame_Btn">
    <property name="geometry">
     <rect>
      <x>499</x>
      <y>340</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>STOP GAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="showing_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select playtime and press START GAME!</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="player1_progressBar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="player2_progressBar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
